src/SBVRType.d.ts

```diff
@@ -0,0 +1,47 @@
+interface SBVRType<I,O> {
+  types: ConcreteTypes
+
+  validate(value: any, required: boolean, cb:Callback<I>): void
```

Comment:
You could use https://github.com/prettier/prettier, it's not exactly the same as our TS guidelines, but nor is this module, and using it on some real project would help to promote it. The benefit is you stop thinking about the code style altogether, the tool is reformatting it for you.